### PR TITLE
chore: add load time metric

### PR DESCRIPTION
**Pull request by dtjones404**

### CODE REVIEW

File: src/tracking.js [hunk 1]
```diff
@@ -123,10 +125,13 @@ export function setupLogger() {
         }`
       )
       .info(cache)
```
Comment (gregjopa) [inline src/tracking.js:127]: Can we also add this metric to the cal log?

```js
 .info(cache, { loadTime })
```

File: src/tracking.js [hunk 2]
```diff
@@ -113,6 +113,8 @@ export function setupLogger() {
     // https://www.paypal.com/buttons/smart
     // https://www.paypal.com/us/gifts/
     const isLoadedInFrame = isPayPalDomain() && window.xprops;
+    const sdkLoadTime =
```
Comment (gregjopa) [inline src/tracking.js:116]: Nit: If we don't need the specific number type check, we can use this syntax to return a string value or undefined.
```suggestion
    const sdkLoadTime = loadTime && String(loadTime.toString);
```

Comment (dtjones404) [reply src/tracking.js]: Ahh load time can be zero though, which breaks the typing

Comment (gregjopa) [reply src/tracking.js:116]: oh good point. I forgot about falsy zero

**Review by gregjopa — APPROVED** [review]

**Pull request merged**